apiv2/src/admin/core/referentiel/classe/useCase/ImporterClasses.ts

```diff
@@ -195,6 +210,17 @@ export class ImporterClasses implements UseCase<ClasseRapport[]> {
         );
     }
 
+    private async resetAffectation(classe: ClasseModel): Promise<void> {
+        if(classe){
+            classe.pointDeRassemblementId = undefined;
+            classe.centreCohesionId = undefined;
+            classe.statutPhase1 = STATUS_PHASE1_CLASSE.WAITING_AFFECTATION;
```

Comment:
```suggestion
            classe.statutPhase1 = STATUS_PHASE1_CLASSE.WAITING_AFFECTATION;
            classe.ligneId = undefined;
```
il faut aussi retirer la ligne de bus si elle existe